movie_ratings.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+"""Movie_Ratings.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1yMfh75GfQG9prr565b-fkb19EQYoiQ70
+"""
+
+import pandas as pd
+import numpy as np
+import matplotlib as plt
+
+imUsingColab = False
+shouldDownload = True # Set this to False after the first run to avoid redownloading the dataset
+if shouldDownload:
```

Comment:
This part isn't necessary

Reply:
We aren't downloading any datasets in this file this should only handle cleaning.